src/content/general/management-clusters/index.md

```diff
@@ -0,0 +1,45 @@
+---
+linkTitle: Management Clusters
+title: Giant Swarm Management Clusters
+description: The role Management Clusters perform in the Giant Swarm architecture and how they are updated.
+weight: 60
+menu:
+  main:
+    identifier: management-clusters
+    parent: general
+last_review_date: 2021-02-17
+user_questions:
+  - How do Giant Swarm use Management Clusters?
+  - How are Management Clusters kept up to date?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-biscuit
+---
+
+## Overview of Management Clusters
+
+As we are fully convinced of Kubernetes as a platform for building platforms, we build all our management clusters based on Kubernetes. Giant Swarm leverages the concept of [“Operators"](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) to control all resources that clusters need as [“Custom Resources”](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
```

Comment:
IMO "operators" should be lowercase (elsewhere as well), but not sure if there's an official style rule for that

Reply:
Yep, please use lowercase. Also, no quotes needed.

Reply:
Great I like that more too. Will fix and remove quotes.